app/controllers/admin/cafe_controller.rb

```diff
@@ -0,0 +1,21 @@
+# encoding:UTF-8
+class Admin::CafeController < ApplicationController
+  skip_authorization
+  before_action :authorize
+
+  def index
+    @cafe_shift_grid = initialize_grid(CafeShift,
+                                       include: :user, order: :start,
+                                      conditions: ['start >= ?', Time.zone.now.beginning_of_day]  )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.<br>Space inside parentheses detected.